src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/BulkAlertService.kt

```diff
@@ -93,7 +93,7 @@ class BulkAlertService(
     chunked(batchSize).flatMap {
       alertRepository.saveAllAndFlush(
         filter { it.activeTo != null }.map {
-          val activeFrom = if (it.willBecomeActive()) LocalDate.now() else it.activeFrom
+          val activeFrom = it.activeFrom
```

Comment:
I would move these into the update statement now that there's no logic associated with calculating it. It might be misleading that it's assigned to a val first.

Reply:
Done.